teamcity: failed test: acceptance

The following tests appear to have failed on master (roachtest): acceptance

You may want to check [for open issues](https://github.com/cockroachdb/cockroach/issues?q=is%3Aissue+is%3Aopen+acceptance).

[#922519](https://teamcity.cockroachdb.com/viewLog.html?buildId=922519):


```
acceptance
--- FAIL: roachtest/acceptance (597.500s)





```

Please assign, take a look and update the issue accordingly.

--- a/build/teamcity-post-failures.py
+++ b/build/teamcity-post-failures.py
@@ -154,7 +154,11 @@ if __name__ == '__main__':
             if len(matches) < 3:
                 matches.append("")
             topname, subname = matches[1], matches[2]
-            if category == "roachtest" and topname == "acceptance" and subname != "":
+            if category == "roachtest" and topname == "acceptance":
+                if subname == "":
+                    # Don't post roachtest failures for the umbrella acceptance test.
+                    # In roachtest, parent tests contain no logic.
+                    continue
                 # Special casing for roachtest so that it doesn't lump any
                 # acceptance failure into an issue that just says "acceptance".
                 topname = subname

